dml_events.go

```diff
@@ -62,8 +60,8 @@ func (e *DMLEventBase) Table() string {
 	return e.table.Name
 }
 
-func (e *DMLEventBase) TableSchema() *schema.Table {
-	return &e.table
```

Comment:
this (`&e.table`) was because we were dereferencing the struct pointer to the `schema.Table` pointer, but is no longer needed because `e.table` is now a struct and not a pointer?